### PR TITLE
Changes to support CSM-523, some PowerScale Replication cleanup

### PR DESCRIPTION
**Pull request by ChristianAtDell**
# Description
Additional text has been added for RG deletion information in PowerScale (see the Deletion subheading in content/docs/replication/architecture/powerscale.md), and IsiPath parameter description has been expanded to make requirements/functionality clearer. 

A child page under Architecture was also created for PowerScale, and PowerScale-specific information has been moved from the PowerScale Deployment page to this child page. The Deployment page was beginning to get cluttered with non-deployment PowerScale information. 

# GitHub Issues
List the GitHub issues impacted by this PR:

| GitHub Issue # |
| -------------- |
|[523](https://github.com/dell/csm/issues/523) |

# Checklist:

- [X] Have you run a grammar and spell checks against your submission?
- [X] Have you tested the changes locally?
- [X] Have you tested whether the hyperlinks are working properly?
- [ ] Did you add the examples wherever applicable?
- [ ] Have you added high-resolution images?

### CODE REVIEW

File: content/docs/replication/deployment/powerscale.md [hunk 1]
```diff
@@ -136,7 +136,7 @@ Let's go through each parameter and what it means:
 
 * `Accesszone` is the name of the access zone a volume can be created in.
 * `AzServiceIP` AccessZone groupnet service IP. It is optional and can be provided if different than the PowerScale cluster endpoint.
-* `IsiPath` is the base path for the volumes to be created on the PowerScale cluster.
+* `IsiPath` is the base path for the volumes to be created on the PowerScale cluster. If not specified in the storage class, the IsiPath defined in the storage array's secret will be used, and if that is not specified, the IsiPath defined in the values.yaml file used for driver installation is used as the lowest-priority. IsiPath between source and target Replication Groups MUST be consistent.
```
Comment (santhoshatdell) [inline content/docs/replication/deployment/powerscale.md:139]: MUST to highlighted **must** in lower case.

File: content/docs/replication/architecture/powerscale.md [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+  Platform-Specific Architecture for CSI PowerScale
+---
+
+### SyncIQ Policy Architecture
+When creating `DellCSIReplicationGroup` (RG) objects on the Kubernetes cluster(s) used for replication, a SyncIQ policy to facilitate this replication is created *only* on the source PowerScale storage array. 
+
+This singular SyncIQ policy on the source storage array and its matching Local Target policy on the target storage array provide information for the RGs to determine their status. Upon creation, the SyncIQ policy is set to a schedule of `When source is modified`. The SyncIQ policy is `Enabled` when the RG is created. The directory that is being replicated is *read-write accessible* on the source storage array, and is restricted to *read-only* on the target. 
+
+### Replication Group Deletion 
+When deleting `DellCSIReplicationGroup` (RG) objects on the Kubernetes cluster(s) used for replication, deletion should only be performed on an empty RG. If there is any user-created or Kubernetes PV-generated data left inside of the replication group, the RG object will be held in a `Deleting` state until all user data has been cleared out.
```
Comment (santhoshatdell) [inline content/docs/replication/architecture/powerscale.md:15]: Should we say 'until all user data has been cleared out **on both source and target storage arrays**' ?

**Review by santhoshatdell — DISMISSED** [review]

**Review by harshaatdell — APPROVED** [review]

File: content/docs/replication/deployment/powerscale.md (hunk 1 above)
Comment (rsedlock1958) [inline content/docs/replication/deployment/powerscale.md]: Remove the comma and place a period (.") after "will be used". Delete "and" and make "if" upper case to start a new sentence,

**Review by rsedlock1958 — COMMENTED** [review]
I have reviewed this. One minor edit is required on line 139.

**Comment by ChristianAtDell** [comment]
Forgot the period in my last commit, sorry. Should be good now.

**Review by harshaatdell — APPROVED** [review]

**Review by rsedlock1958 — APPROVED** [review]
I have reviewed this. It looks gfine.

**Pull request merged**